src/System.Numerics.Vectors/ref/System.Numerics.Vectors.cs

```diff
@@ -471,6 +471,8 @@ public partial struct Vector<T> : System.IEquatable<System.Numerics.Vector<T>>,
     {
         private int _dummyPrimitive;
 #if HAS_SPAN
```

Comment:
This predates your PR, but I thought we generally separated .NET Core additions like this into a separate .netcoreapp.cs file when we need to build the ref for both old and new surface areas?

Reply:
I think you're right. This is the only place across corefx where I see `HAS_SPAN` in any reference assembly project.